.github/workflows/TestSuite-Release.yml

```diff
@@ -1,8 +1,5 @@
 name: Test Suite - Release
 on:
-  release:
-    types:
-      - published
```

Comment:
When the release is already published is a little late to run this workflow.
We have to run it manually to make sure that the release is all good.